src/components/Anchor/Anchor.js

```diff
@@ -71,7 +75,9 @@ const inferType = (children) => {
   }
 
   return 'wrap';
-}
+};
+
+const isExternal = (to) => /^(https?)*:\/\//.test(to);
```

Comment:
It matches `https://foo` and `http://foo` (since the `?` makes the `s` optional). I just updated it to also match `//foo` for protocol-irrelevant links.